showcase/app/templates/components/time.hbs

```diff
@@ -0,0 +1,93 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Time Component"}}
+
+<Shw::Text::H1>Time</Shw::Text::H1>
+
+{{! Time component does not include styling so we do not test in Percy at this time }}
+
+<section>
+  <Shw::Text::H2>Has tooltip</Shw::Text::H2>
+
+  <Shw::Flex @gap="2rem" as |SF|>
+    <SF.Item @label="With tooltip (default)">
+      <Hds::Time @date="05 September 2018 14:48" />
+    </SF.Item>
+
+    <SF.Item @label="Without tooltip">
+      <Hds::Time @date="05 September 2018 14:48" @hasTooltip={{false}} />
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H2>Display</Shw::Text::H2>
+
+  <p>Note: “examples” are temporary</p>
+
+  <Shw::Flex @gap="4rem 9rem" as |SF|>
+    <SF.Item @label="Default with display unset">
+      <Hds::Time @date="05 September 2018 14:07:32" @isOpen={{true}} />
+    </SF.Item>
+
+    {{#each @model.DISPLAYS as |display|}}
+      <SF.Item @label={{display}}>
+        <Hds::Time @date="05 September 2018 14:07:32" @display={{display}} @isOpen={{true}} />
+      </SF.Item>
+    {{/each}}
+  </Shw::Flex>
+</section>
+
+<Shw::Divider {{style marginTop="6rem"}} @level="2" />
+
+<section>
+  <Shw::Text::H3>Near time examples using “relative” display type</Shw::Text::H3>
+
+  <Shw::Flex @gap="2rem" as |SF|>
+    {{#each this.nearTimeDates as |date|}}
+      <SF.Item @label={{date}}>
+        <Hds::Time @date={{date}} @display="relative" />
+      </SF.Item>
+    {{/each}}
+  </Shw::Flex>
+</section>
+
+<Shw::Divider />
+
+<section>
+  {{! Note: Skipping Percy tests for this section as times continually update }}
```

Comment:
Suggestion: is this comment still needed if the whole file isnt covered by percy?

Reply:
If someone comes into the file to edit they may wonder why the data attribute is missing

Reply:
Oh wait you're right this was the old comment not the new one I added... I'll delete.